contracts/CPoRDelegator.sol

```diff
@@ -0,0 +1,477 @@
+pragma solidity ^0.5.16;
+
+import "./CPoR.sol";
+
+/**
+ * @title Compound's CPoRDelegator Contract
+ * @notice CPoRTokens which wrap an EIP-20 underlying and delegate to an implementation
+ *  and checks a proof-of-reserves feed before minting.
+ * @author Compound
+ */
+contract CPoRDelegator is CPoR, CDelegatorInterface {
```

Comment:
Ideally this should implement `CTokenInterface, CErc20Interface, CPoRInterface, CDelegatorInterface` but the order is apparently wrong, I'm guessing because of the way Solidity linearises multiple levels of inheritance

Reply:
`CPoRDelegator` is exactly like `CErc20Delegator`, except that it implements the storage slots required by `CPoRInterface`, which is implemented by `CPoRDelegate`, which this contract does delegatecalls to.